src/main/java/bobby/ui/Bobby.java

```diff
@@ -0,0 +1,366 @@
+package bobby.ui;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * Bobby is a chatbot that can manage a list of tasks, ToDos, Deadlines, and Events.
+ * It can mark tasks as done, unmark, delete and add tasks.
+ */
+public class Bobby {
+
+    /**
+     * Greeting message displayed when the bot starts
+     */
+    private static final String GREETING = "Hello! I'm Bobby\n"
```

Comment:
Maybe the constants GREETING and EXIT could be renamed to something more descriptive, like GREETING_MESSAGE and EXIT_MESSAGE?